src/main/java/com/whalespottingjava/models/database/Sighting.java

```diff
@@ -0,0 +1,62 @@
+package com.whalespottingjava.models.database;
+
+import java.sql.Date;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "whale_sighting")
+public class Sighting {
+    private static final String ID_SEQUENCE = "sighting_id_sequence";
+
+    @Id
+    @SequenceGenerator(name = ID_SEQUENCE, sequenceName = ID_SEQUENCE, allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = ID_SEQUENCE)
+    private Long id;
+
+    @Column(nullable = false)
+    private int member_id;
+
+    private Date date;
+    private double latitude;
+    private double longitude;
+    private Boolean approved;
```

Comment:
Prefer booleans to be named with "is" or "has" or some other thing that makes it seem like a statement
```suggestion
    private Boolean isApproved;
```
(You'll need to change this throughout the file)

Reply:
Plan to fix in a separate PR